{{cookiecutter.project_slug}}/Makefile

```diff
@@ -285,12 +292,20 @@ train: _check_setup upload-code upload-config   ### Run a training job
 		bash -c 'cd $(PROJECT_PATH_ENV) && $(TRAIN_CMD)'
 
 .PHONY: kill-train
-kill-train:  ### Terminate the training job
-	$(NEURO) kill $(TRAIN_JOB)
+kill-train:  ### Terminate the training job (set up env var 'RUN' to specify the training job)
+	$(NEURO) kill $(TRAIN_JOB)-$(RUN)
+
+.PHONY: kill-train-all
+kill-train-all:  ### Terminate all training jobs you have submitted (set up env var 'RUN' to specify the training job)
+ifneq ($(wildcard $(_TRAIN_JOB_FILE)),)
+	$(NEURO) kill $$(cat $(_TRAIN_JOB_FILE)) && rm $(_TRAIN_JOB_FILE)
```

Comment:
Note: currently, `neuro kill` always returns zero exit code, so the file will always be removed